packages/cli-lib/modules.js

```diff
@@ -58,6 +58,10 @@ const isModuleFolderChild = pathInput => {
   const { length } = pathParts;
   // Not a child path?
   if (length <= 1) return false;
+  // Check if we should ignore this file
+  if (ignoreLocales && pathParts.find(part => part === '_locales')) {
```

Comment:
Are there other folders/files we should be ignoring?

Reply:
I asked Mitchell about that yesterday, and it sounds like this is the only one we should ignore.